subsystem/operators_test.go

```diff
@@ -217,13 +218,126 @@ var _ = Describe("Operators endpoint tests", func() {
 			return cluster
 		}
 
+		It("LSO as ODF dependency on P arch", func() {
+			// Register cluster with ppc64le CPU architecture
+			cluster := registerNewCluster(
+				"4.13.0",
+				models.ClusterHighAvailabilityModeFull,
+				nil,
+				swag.String(models.ClusterCPUArchitectureS390x),
+			)
+			Expect(cluster.Payload.CPUArchitecture).To(Equal(models.ClusterCPUArchitectureMulti))
+			Expect(len(cluster.Payload.MonitoredOperators)).To(Equal(1))
+
+			// Register infra-env with ppc64le CPU architecture
+			infraEnvParams := installer.RegisterInfraEnvParams{
+				InfraenvCreateParams: &models.InfraEnvCreateParams{
+					Name:             swag.String("infra-env-1"),
+					OpenshiftVersion: "4.13.0",
+					ClusterID:        cluster.Payload.ID,
+					PullSecret:       swag.String(fmt.Sprintf(psTemplate, FakePS2)),
+					SSHAuthorizedKey: swag.String(sshPublicKey),
+					CPUArchitecture:  models.ClusterCPUArchitectureS390x,
```

Comment:
the test name talking about P but the arch is Z

Reply:
Fixed